app/lib/context-helper.js

```diff
@@ -5,12 +5,14 @@ const {
   endemicsClaim: {
     latestEndemicsApplication: latestEndemicsApplicationKey,
     latestVetVisitApplication: latestVetVisitApplicationKey,
-    previousClaims: previousClaimsKey
+    previousClaims: previousClaimsKey,
+    reference: referenceKey
   }
 } = require('../session/keys')
 const { getClaimsByApplicationReference } = require('../api-requests/claim-service-api')
-const { getEndemicsClaim } = require('../session')
+const { getEndemicsClaim, clearEndemicsClaim } = require('../session')
 const { claimType } = require('../constants/claim')
+const createClaimReference = require('../lib/create-temp-claim-reference')
```

Comment:
Be nice if this file had a named export which matched it's file name. Currently we have a file called `create-temp-claim-reference` which exports `generateRandomID` which we are importing as `createClaimReference`.

Reply:
much better 👍